.jenkins/caffe2/build.sh

```diff
@@ -140,8 +150,15 @@ CMAKE_ARGS+=("-DONNX_NAMESPACE=ONNX_NAMESPACE_FOR_C2_CI")
 ${CMAKE_BINARY} "${ROOT_DIR}" ${CMAKE_ARGS[*]} "$@"
 
 # Build
+# sccache will fail for CUDA builds if all cores are used for compiling
```

Comment:
Do you know why?

Reply:
I suspect that it's because `sccache nvcc` is using `sccache gcc` as host compiler by default, but if all cores are already used up by the `sccache nvcc` invocations, there will be no core available to the `sccache gcc` invocations and the compilation will be stuck. As comparison, on Windows we explicitly specify `cl.exe` as CUDA_HOST_COMPILER and this issue doesn't appear.

There is probably a way to fix it by forcing NVCC to use `/usr/bin/cc` as host compiler regardless of `-ccbin` settings, but I haven't figured out a nice way to do so yet.